TestFoundation/TestURL.swift

```diff
@@ -240,6 +256,42 @@ class TestURL : XCTestCase {
     static let gDirectoryDoesNotExistPath = gBaseTemporaryDirectoryPath + gDirectoryDoesNotExistName
 
     static func setup_test_paths() -> Bool {
+#if os(Windows)
```

Comment:
Outside of testing `FileManager`, this should really just use `FileManager` calls now that the class is basically entirely implemented.